client/templates/userprofile/settings.html

```diff
@@ -35,7 +37,7 @@ <h2 class="userprofile-sub-title">
                        value="{{profile.username}}">
             </div>
         <div class="form-group {{#if emailerror}}has-error{{/if}}">
-            <p class="profile_errors">{{emailerror}}</p>
+            <p class="email_errors errors">{{emailerror}}</p>
```

Comment:
To increase readability, use one space around helpers within double handlebars:
* {{wrong}}
* {{ right }}